frontend/yarn-audit-known-issues

```diff
@@ -89,6 +89,10 @@
ebpack-plugin>cacache>tar","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"6.1.0","paths":["react-scripts>terser-webpack-plugin>cacache>tar"]},{"version":"2.2.2","paths":["react-js-pagination>tar"]}],"id":1770,"created":"2021-08-03T18:11:06.582Z","updated":"2021-08-03T19:07:08.152Z","deleted":null,"title":"Arbitrary File Creation/Overwrite due to insufficient absolute path sanitization","found_by":{"link":"","name":"Anonymous","email":""},"reported_by":{"link":"","name":"Anonymous","email":""},"module_name":"tar","cves":["CVE-2021-32804"],"vulnerable_versions":"<3.2.2 || >=4.0.0 <4.4.14 || >=5.0.0 <5.0.6 || >=6.0.0 <6.1.1","patched_versions":">=3.2.2 <4.0.0 || >=4.4.14  <5.0.0 || >=5.0.6 <6.0.0 || >=6.1.1","overview":"The `tar` package has a high severity vulnerability before versions 3.2.2, 4.4.14, 5.0.6, and 6.1.1.\n\n### Impact\n\nArbitrary File Creation, Arbitrary File Overwrite, Arbitrary Code Execution\n\n`node-tar` aims to prevent extraction of absolute file paths by turning absolute paths into relative paths when the `preservePaths` flag is not set to `true`. This is achieved by stripping the absolute path root from any absolute file paths contained in a tar file. For example `/home/user/.bashrc` would turn into `home/user/.bashrc`. \n\nThis logic was insufficient when file paths contained repeated path roots such as `////home/user/.bashrc`. `node-tar` would only strip a single path root from such paths. When given an absolute file path with repeating path roots, the resulting path (e.g. `///home/user/.bashrc`) would still resolve to an absolute path, thus allowing arbitrary file creation and overwrite. \n\n### Workarounds\n\nUsers may work around this vulnerability without upgrading by creating a custom `onentry` method which sanitizes the `entry.path` or a `filter` method which removes entries with absolute paths.\n\n```js\nconst path = require('path')\nconst tar = require('tar')\n\ntar.x({\n  file: 'archive.tgz',\n  // either add this function...\n  onentry: (entry) => {\n    if (path.isAbsolute(entry.path)) {\n      entry.path = sanitizeAbsolutePathSomehow(entry.path)\n      entry.absolute = path.resolve(entry.path)\n    }\n  },\n\n  // or this one\n  filter: (file, entry) => {\n    if (path.isAbsolute(entry.path)) {\n      return false\n    } else {\n      return true\n    }\n  }\n})\n```\n\nUsers are encouraged to upgrade to the latest patch versions, rather than attempt to sanitize tar input themselves.","recommendation":"Upgrade to version 3.2.2, 4.4.14, 5.0.6, 6.1.1 or later","references":"- [GitHub Advisory](https://github.com/npm/node-tar/security/advisories/GHSA-3jfq-g458-7qm9)\n- [CVE](https://nvd.nist.gov/vuln/detail/CVE-2021-32804)\n- [Related but distinct advisory involving symlinks](https://www.npmjs.com/advisories/1771)","access":"public","severity":"high","cwe":"CWE-22","metadata":{"module_type":"","exploitability":8,"affected_components":""},"url":"https://npmjs.com/advisories/1770"}}}
```

Comment:
Could you please use what's in master here? I think using a single PR(https://github.com/adhocteam/Head-Start-TTADP/pull/378) to clean up the yarn audit issue is cleaner.

Reply:
Ok I have merged the audit vuln file from main.